fakeredis.py

```diff
@@ -1273,6 +1273,42 @@ def transaction(self, func, *keys):
                     continue
         raise redis.WatchError('Could not run transaction after 5 tries')
 
+    # HYPERLOGLOG COMMANDS
+    def pfadd(self, name, *values):
+        "Adds the specified elements to the specified HyperLogLog."
+        # Simulate the behavior of HyperLogLog by using SETs underneath to
+        # approximate the behavior.
+        result = self.sadd(name, *values)
+
+        # Per the documentation:
+        # - 1 if at least 1 HyperLogLog internal register was altered. 0 otherwise.
+        return 1 if result > 0 else 0
+
+    def pfcount(self, name):
+        """
+        Return the approximated cardinality of
+        the set observed by the HyperLogLog at key(s).
+        """
+        return self.scard(name)
+
+        # This will be the proper implementation, once `fakeredis` uses a
+        # post-2.10.3 release of `redis-py`.
+        # https://github.com/andymccurdy/redis-py/commit/6935a30c680a81659b2e02b9f9220517cdfd637b
+        # def pfcount(self, *sources):
```

Comment:
I would do it if I could, but the ```redis-py``` package hasn't released anything after 2.10.3 yet, so I have no later version to point to. (No release in that project since 14-Aug-2014!)